test/Microsoft.AspNet.Antiforgery.Test/DefaultAntiforgeryTokenStoreTest.cs

```diff
@@ -220,6 +253,7 @@ public void GetCookieToken_CookieIsValid_ReturnsToken()
         {
             // Arrange
             var httpContext = new DefaultHttpContext();
+            httpContext.Request.ContentType = "application/x-www-form-urlencoded";
```

Comment:
This isn't strictly needed because `HasFormContentType` checks for `Form != null` but it seemed good for clarity.